<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BDC4-L-PF1V7L8G" name="com.alexion.testRunner.testRunner" tests="1" failures="0" timestamp="16 Jun 2020 12:44:19 GMT" time="0.000" errors="1">
  <testcase name="runScenario" time="0.000" classname="com.alexion.testRunner.testRunner">
    <error type="java.lang.RuntimeException" message="java.lang.IllegalArgumentException: Not a file or directory: C:\Users\naveen.kumar.meduri\gitCucumberGit\com.alexion.NewsandMedia\src\test\resources\contactus.feature">
      <![CDATA[java.lang.RuntimeException: java.lang.IllegalArgumentException: Not a file or directory: C:\Users\naveen.kumar.meduri\gitCucumberGit\com.alexion.NewsandMedia\src\test\resources\contactus.feature
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:163)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1274)
	at org.testng.internal.Invoker.createParameters(Invoker.java:989)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1079)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.IllegalArgumentException: Not a file or directory: C:\Users\naveen.kumar.meduri\gitCucumberGit\com.alexion.NewsandMedia\src\test\resources\contactus.feature
	at cucumber.runtime.io.FileResourceIterator$FileIterator.<init>(FileResourceIterator.java:63)
	at cucumber.runtime.io.FileResourceIterator.<init>(FileResourceIterator.java:28)
	at cucumber.runtime.io.FileResourceIterator.createFileResourceIterator(FileResourceIterator.java:14)
	at cucumber.runtime.io.FileResourceIterable.iterator(FileResourceIterable.java:19)
	at cucumber.runtime.model.FeatureLoader.loadFromFeaturePath(FeatureLoader.java:31)
	at cucumber.runtime.model.FeatureLoader.load(FeatureLoader.java:23)
	at cucumber.runtime.FeaturePathFeatureSupplier.get(FeaturePathFeatureSupplier.java:33)
	at io.cucumber.testng.TestNGCucumberRunner.getFeatures(TestNGCucumberRunner.java:116)
	at io.cucumber.testng.TestNGCucumberRunner.provideScenarios(TestNGCucumberRunner.java:98)
	at io.cucumber.testng.AbstractTestNGCucumberTests.scenarios(AbstractTestNGCucumberTests.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	... 20 more
]]>
    </error>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.alexion.testRunner.testRunner -->
